ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -28,6 +28,15 @@
     @{
         var Event = (Model as ServerCore.ModelBases.EventSpecificPageModel)?.Event;
         var EventRole = (Model as ServerCore.ModelBases.EventSpecificPageModel)?.EventRole;
+        var AdminRoute = "/EventSpecific/Index";

```

Comment:
Behold the fancy "??" operator that condenses each of these to one line:

var AdminRoute = ViewData["AdminRoute"] as string ?? "/EventSpecific/Index";

"??" means "use the thing to my left if it is non-null, otherwise use the thing to my right".

Reply:
Oh my goodness this is awesome - so much cleaner :D Updated :)